### ISSUE TITLE
WoRMS Bryozoa missing names

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#467]
When looking into the [not interpreted and nameID invalid issues](https://www.checklistbank.org/dataset/1081/issues), I found that there are name relations defined that point to names which are not included in the export.

For example [Lichenalia circincta Hall, 1883](https://www.marinespecies.org/aphia.php?p=taxdetails&id=1419422) is the type species of genus [Selenopora](https://www.marinespecies.org/aphia.php?p=taxdetails&id=1373234). This is also given as such in this name relation record: https://www.checklistbank.org/dataset/1081/verbatim/1274

Unfortunately only Selenopora exists in the Name.txt file, Lichenalia circincta is missing:

 - https://www.checklistbank.org/dataset/1081/name/urn:lsid:marinespecies.org:taxname:1419422
 - https://www.checklistbank.org/dataset/1081/name/urn:lsid:marinespecies.org:taxname:1373234

**Comment by claudenozeres** [comment]
For WoRMS, this example is a fossil, so is hidden by default? Perhaps the fetch query could not work for that reason, e.g., https://www.checklistbank.org/dataset/1081/name/urn:lsid:marinespecies.org:taxname:1419422
vs: https://www.marinespecies.org/aphia.php?p=taxdetails&id=1419422 (had to override to obtain?).

**Comment by mdoering** [comment]
Yes, the name 1419422 is missing from the archive and still linked from some records:
https://www.checklistbank.org/dataset/1081/verbatim?q=1419422&type=col%3AName

But it also seems to be hidden in WoRMS, maybe thats the reason it does not get exported?
